src/apps/filters/h3.c

```diff
@@ -75,7 +118,8 @@ H3Error cellToLatLngCmd(int argc, char *argv[]) {
     return E_SUCCESS;
 }
 
-H3Error latLngToCellCmd(int argc, char *argv[]) {
+SUBCOMMAND(latLngToCell,
+           "Convert degrees latitude/longitude coordinate to an H3 cell") {
```

Comment:
I haven't looked at the CLI closely before, so apologies if this is a dumb question: I'm guessing there's no easy way in the CLI to get these functions to be inverses of each other (`latLngToCell` and `cellToLatLng`)?

Reply:
I think a WKT input does make sense, as well, but it just hasn't been done, yet. I'm thinking a `-w / --wkt` arg would probably fit.

Reply:
I think the reason why we didn't do that at first is because it could be confusing what to do if the WKT text is not just a `POINT`. Should it automatically compute cells for each point provided if it's passed a `LINESTRING`? Making decisions on the "right" behavior in situations like that was basically punted for later.